src/main/java/com/epam/ms/util/UserServiceMediator.java

```diff
@@ -0,0 +1,50 @@
+package com.epam.ms.util;
+
+import com.epam.ms.util.dto.User;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.MediaType;
+import org.springframework.stereotype.Component;
+import org.springframework.web.reactive.function.client.WebClient;
+
+import java.util.List;
+
+/**
+ * The class is used to send request to the user-service
+ */
+@Component
+public class UserServiceMediator {
+    private static final String HOST_URL = "http://localhost:%s";
```

Comment:
`HOST_URL`
It can also be defined in `application.yml`
Example:
```
clients:
  user-service:
      url: http://localhost:8080
```
then, you can have [different profiles](https://www.baeldung.com/spring-profiles) for different environment (local, dev, qa, prod):
`application-qa.yml`:

```
clients:
  user-service:
      url: http://my-server-qa:8080
```

Reply:
Thank you, done